robert_beverage/scripts/generate_qr_code

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env python
 import rospy
```

Comment:
please remove this dependency and ros node initialization code. For this utility you do not need ROS node.